src/data/02-programming-languages/fibjs.yml

```diff
@@ -0,0 +1,11 @@
+name: 'FibJS'
+homepageURL: 'https://fibjs.org'
+repoURL: 'https://github.com/fibjs/fibjs'
+portingEfforts:
+  - authors: ['pinelliaw']
+    desc: ''
+    link: 'https://github.com/fibjs/fibjs'
```

Comment:
This should point to the actual issue tracking the porting effort, or the commit that marks the finish of the work. Otherwise it should be left empty to be consistent with the other records.

But for now I think fibjs/fibjs#770 should do...